projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -85,6 +90,52 @@ function NewsletterAccessSetupNudge( { stripeConnectUrl, isStripeConnected, hasN
 	}
 }
 
+function TierSelector( { onChange } ) {
+	// TODO: figure out how to handle different currencies
+	const products = useSelect( select => select( membershipProductsStore ).getProducts() )
+		.filter( product => product.subscribe_as_site_subscriber && product.interval === '1 month' )
+		.sort( ( p1, p2 ) => Number( p2.price ) - Number( p1.price ) );
+
+	// Find the current tier meta
+	const postType = useSelect( select => select( editorStore ).getCurrentPostType(), [] );
+	// Destructure the tierId from the meta (set tierId using the META_NAME_FOR_POST_TIER_ID_SETTINGS constant)
+	const [ { [ META_NAME_FOR_POST_TIER_ID_SETTINGS ]: tierId } ] = useEntityProp(
+		'postType',
+		postType,
+		'meta'
+	);
+
+	// Tiers don't apply if less than 2 products (this is called here because
+	// the hooks have to run before any early returns)
+	if ( products.length < 2 ) {
+		return;
+	}
+
+	return (
+		<div className="editor-post-tiers">
+			{ products.map( product => (
+				<div key={ product.id }>
+					<input
```

Comment:
Would be good to use the core components for basic form elements if possible. See [`RadioControl`](https://wordpress.github.io/gutenberg/?path=/docs/components-radiocontrol--docs).

Fine to do in a follow-up PR after merging this.

Reply:
We'll do it in this one. @n3f Can you do this, please?

Reply:
Fix here: https://github.com/Automattic/jetpack/pull/32928/files